packages/template-retail-react-app/package.json

```diff
@@ -97,7 +97,7 @@
     },
     {
       "path": "build/vendor.js",
-      "maxSize": "320 kB"
+      "maxSize": "370 kB"
```

Comment:
:oof:

Reply:
@wjhsf  We have a [ticket](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001e81pJYAQ/view) to address that. let's focus on steel thread for now.